### DISCUSSION TITLE
Double space after a period

### DISCUSSION THREAD

**Discussion by debrandell** [opensteno/plover#1593] [Q&A and Support] [answered]
The company I will be doing transcription work for requires a double space after periods and colons, rather than a single space.  I haven't been able to find out how to incorporate the second space, so if anyone has a clue how to do this, I'd be very grateful for the info. Thank you!

**Comment by aerickt** [comment, marked as answer]
For period
```
{.}{^  ^}{-|}
```

For colon
```
{:}{^  ^}
```

These should work; you just have to add your own outlines with the "add translation" tool in Plover.

**Comment by debrandell** [comment]
Thank you!! I'll input these and give it a try. I appreciate your quick response!

All the best,
Deb Brandell

Sent using myEarthLink

On Tue Feb 28 21:41:47 EST 2023 Aerick wrote:

For period

{.}{^ ^}{-|}

For colon

{:}{^ ^}

These should work; you just have to add your own outlines with the "add translation" tool in Plover.

—
Reply to this email directly, view it on GitHub (https://github.com/openstenoproject/plover/discussions/1593#discussioncomment-5161855), or unsubscribe (https://github.com/notifications/unsubscribe-auth/A6GEE6OSOHLL35XXCTAHKI3WZ2ZOVANCNFSM6AAAAAAVLOA6SA).
You are receiving this because you authored the thread.Message ID:

<openstenoproject/plover/repo-discussions/1593/comments/

5161855 (tel:5161855)

@github.com>

**Comment by debrandell** [comment]
Just to clarify, is the "add translation" tool something different than the Dictionary Editor? I have just begun using Plover so I'm not completely familiar with it. It's great, by the way! I did enter these commands in the Dictionary Editor and it didn't change anything in the transcription; still only one space. So I'll see if I can find where the add translation tool is. Thanks!

Sent using myEarthLink

On Tue Feb 28 21:41:47 EST 2023 Aerick wrote:

For period

{.}{^ ^}{-|}

For colon

{:}{^ ^}

These should work; you just have to add your own outlines with the "add translation" tool in Plover.

—
Reply to this email directly, view it on GitHub (https://github.com/openstenoproject/plover/discussions/1593#discussioncomment-5161855), or unsubscribe (https://github.com/notifications/unsubscribe-auth/A6GEE6OSOHLL35XXCTAHKI3WZ2ZOVANCNFSM6AAAAAAVLOA6SA).
You are receiving this because you authored the thread.Message ID:

<openstenoproject/plover/repo-discussions/1593/comments/

5161855 (tel:5161855)

@github.com>

**Reply by debrandell** [reply]
Got it!  Thank you!!!
